Tasks/AppCenterTest/package.json

```diff
@@ -18,7 +18,7 @@
   "homepage": "https://github.com/microsoft/vsts-tasks",
   "dependencies": {
     "glob": "^7.0.6",
-    "mobile-center-cli": "0.12.0",
+    "appcenter-cli": "^1.0.0",
```

Comment:
Are you sure you don't want to pin to a specific version?

Reply:
I was wondering about that, I think I will pin it to be safe :)